packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -220,6 +220,24 @@ export class Account {
   ): Promise<AccountHistory[]> {
     return await this.client.call('listaccounthistory', [owner, options], 'number')
   }
+
+  /**
+   * Creates (and submits to local node and network) a transfer transaction from the wallet UTXOs to specfied account.
+   *
+   * @param {string} address
+   * @param {number} amount
+   * @param {UtxosToAccountOptions} [options]
+   * @param {UtxosToAccountUTXO[]} [options.utxos] utxos array of specific UTXOs to spend
+   * @param {string} [options.utxos.txid]
+   * @param {number} [options.utxos.vout]
+   * @return {Promise<string>}
+   */
+  async utxosToAccount (address: string, amount: number, options: UtxosToAccountOptions = {}): Promise<string> {
+    const { utxos = [] } = options
+    const payload: { [key: string]: string } = {}
+    payload[address] = `${amount.toString()}@0`
```

Comment:
Don't hardcode the tokenId
see https://github.com/DeFiCh/ain/blob/df796cb7d6dbbe0ed7d051e79db0f36630f77afb/src/masternodes/rpc_accounts.cpp#L509

Reply:
The commit seems polluted but actually, it is not. I separated the unit test into two (Added listing & transfer describes). And this added space to most of the existing lines of code.  
Intellij has a more powerful file comparison tool. See https://drive.google.com/file/d/1IX1Qij1o_n3nSiVMuxa-9tiQLcrZfoaG/view?usp=sharing
You can see that I just changed the code I needed

For tokenId: Ya, I will take a look at it.

Appreciate if could find more problem

Reply:
I have spent hours passing in tokens as parameter and I keep getting `Insufficient Fund `error. I talked to Peter just now. He said that the comment is wrong and should be updated.  He told me that utxosToAccount rpc ONLY accepts DFI coin as parameter,.
Let me update my code again. I will push later and you may review my code again